### PR TITLE
Add "chemicals" package

### PR DESCRIPTION
**Pull request by volpatto**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by volpatto** [comment]
@CalebBell could please confirm here if you want to maintain the `chemicals` alongside me? I have noticed that `chemicals` has another developer. Should we include him?

Thanks!

File: recipes/chemicals/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - chemicals
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/CalebBell/chemicals
+  summary: Chemical properties component of Chemical Engineering Design Library (ChEDL)
```
Comment (volpatto) [inline recipes/chemicals/meta.yaml:38]: Sorry to bother you again, @CalebBell. Is this summary accurate enough? It looks like another package you have. Please feel free to suggest improvements.

**Comment by CalebBell** [comment]
Hi Diego,
Awesome work! Everything looks wonderful. I will try to come up with a better summary soon but this is good for now. Yoel Cortes has been helping quite a lot and he helped me refactor the library to be more maintainable. Let's add @yoelcortes to this list. 

I still don't have a concrete plan for upgrading thermo. The biggest barriers for me are that my local copy has tons of new features but the code quality has decreased and I still don't have a set of APIs that will be maintainable long term. Software is hard.

Thank you for putting this package together!
Caleb

**Review by marcelotrevisani — APPROVED** [review]

**Comment by yoelcortes** [comment]
Thanks for including me! I would suggest changing the name to the following (while its still early):

chemicals: Chemical database of Chemical Engineering Design Library (ChEDL)

Kinda similar to `thermo`'s but at least it makes sense.

**Comment by volpatto** [comment]
> Hi Diego,
> Awesome work! Everything looks wonderful. 

Thanks!! You did the work, I am just helping to spread it!

> I will try to come up with a better summary soon but this is good for now. Yoel Cortes has been helping quite a lot and he helped me refactor the library to be more maintainable. Let's add @yoelcortes to this list.

Great! I just checked that you have modified the feedstock repo. Thanks!


> I still don't have a concrete plan for upgrading thermo. The biggest barriers for me are that my local copy has tons of new features but the code quality has decreased and I still don't have a set of APIs that will be maintainable long term. Software is hard.

Yeah, I definitely understand how you feel. With the software growing, it becomes really hard to maintain. And it demands time. No worries!

**Comment by volpatto** [comment]
> Thanks for including me! 

Great news, Yoel. Let's update the chemicals-feedstock repo.

> I would suggest changing the name to the following (while its still early):
> chemicals: Chemical database of Chemical Engineering Design Library (ChEDL)

Sounds good to me.

**Pull request merged**